charts/csi-powerstore/values.yaml

```diff
@@ -23,7 +23,25 @@
 driverName: "csi-powerstore.dellemc.com"
 
 # Driver version required to pull the latest driver image
-version: "v2.7.0"
+version: "v2.8.0"
+
+images:
+  # "driver" defines the container image, used for the driver container.
+  driver: dellemc/csi-powerstore:v2.8.0
+  # CSI sidecars
+  attacher: registry.k8s.io/sig-storage/csi-attacher:v4.3.0
+  provisioner: registry.k8s.io/sig-storage/csi-provisioner:v3.5.0
+  snapshotter: registry.k8s.io/sig-storage/csi-snapshotter:v6.2.2
+  resizer: registry.k8s.io/sig-storage/csi-resizer:v1.8.0
+  registrar: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.8.0
+  healthmonitor: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.9.0
+
+  # CSM sidecars
+  replication: dellemc/dell-csi-replicator:v1.5.0
```

Comment:
Update the image for dell-csi-replicator to v1.6.0

Reply:
Done